### PR TITLE
IT-5839 | OverlayHeader | Enable ellipsis on title

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

**Review by rogatty — APPROVED** [review]

File: lib/js/components/Headers/OverlayHeader/OverlayHeader.vue [hunk 1]
```diff
@@ -100,13 +101,15 @@
...
 @import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/shadows';
 @import '../../../../styles/settings/typography/tokens';
 @import '../../../../styles/mixins/flex-overflow-mask';
 
 .ds-overlayHeader {
 	align-items: center;
 	background: $color-neutral-background;
 	border-bottom: 2px solid $color-neutral-border-ghost;
+	box-shadow: $shadow-s;
```
Comment (rogatty) [inline lib/js/components/Headers/OverlayHeader/OverlayHeader.vue:112]: Now we need to remove https://github.com/bethinkpl/wnl-platform/blob/c3ebeb5abb71c3fb77b5588c4d382be21020cc17/resources/assets/js/modules/multistepProblems/components/MultistepProblemOverlayView.vue#L88
It looks bad with both applied at once:
Screenshot 2024-07-26 at 09 18 08

Comment (rogatty) [reply lib/js/components/Headers/OverlayHeader/OverlayHeader.vue:112]: Plus https://github.com/bethinkpl/wnl-platform/blob/0a7fc91265abaf83ab5393159cce7c94c3fdf3ac/resources/assets/js/modules/assessments/components/plans/editor/AssessmentsPlanEditor.vue#L195

**Review by rogatty — APPROVED** [review]

File: lib/js/components/Headers/OverlayHeader/OverlayHeader.vue (hunk 1 above)
Comment (kvas-damian) [reply lib/js/components/Headers/OverlayHeader/OverlayHeader.vue:112]: https://github.com/bethinkpl/wnl-platform/pull/5102/commits/0ca3b6d214bcf3a286814f647c4b18e1b7fa0999

**Pull request merged**